### PR TITLE
Add index.html

### PR DESCRIPTION
**Pull request by Turtleb01**

### CODE REVIEW

**Review by ahnl — APPROVED** [review]
Looks quite pretty and good to me. Thank you for your contribution, it is surely a useful one! :)

I'm thinking whether it would be easier to spot beta versions if the state would be appended to the version name field: for example `ABITTI21233 (Beta)`. Additionally, what would you think if we renamed *Beta* to *Unreleased*, as it could be slightly more descriptive naming?

**Review by ahnl — CHANGES_REQUESTED** [review]
Actually I just noticed a minor thing! Could you please move index.html to the docs/ folder, as the GitHub Pages is serving files from there?

**Comment by Esinko** [comment]
_Unreleased_ is better than _beta_ in my opinion. As the versions we currently know as _beta_ versions were never meant to be publicly tested, thus calling them test versions of sorts doesn't make that much sense. Though regardless of this fact, the current _beta_ versions exist for YTL's internal testing purposes and are not real releases... so calling them _unreleased_ versions would not make that much sense either.
In conclusion: Who cares? Looks good to me!

**Review by developerfromjokela — APPROVED** [review]
Looks cool! For now, this looks good. Maybe changing the design in future 😄

**Comment by developerfromjokela** [comment]
> _Unreleased_ is better than _beta_ in my opinion. As the versions we currently know as _beta_ versions were never meant to be publicly tested, thus calling them test versions of sorts doesn't make that much sense. Though regardless of this fact, the current _beta_ versions exist for YTL's internal testing purposes and are not real releases... so calling them _unreleased_ versions would not make that much sense either.
> In conclusion: Who cares? Looks good to me!

Not exactly true, they publish in their "blog" about upcoming beta versions.
https://www.abitti.fi/blogi/2020/10/uusi-abitti-versio-beta-testikaytossa/

**Comment by Esinko** [comment]
> > _Unreleased_ is better than _beta_ in my opinion. As the versions we currently know as _beta_ versions were never meant to be publicly tested, thus calling them test versions of sorts doesn't make that much sense. Though regardless of this fact, the current _beta_ versions exist for YTL's internal testing purposes and are not real releases... so calling them _unreleased_ versions would not make that much sense either.
> > In conclusion: Who cares? Looks good to me!
> 
> Not exactly true, they publish in their "blog" about upcoming beta versions.
> https://www.abitti.fi/blogi/2020/10/uusi-abitti-versio-beta-testikaytossa/

Hmm. Well in that case _beta_ is the way to go here.

**Review by ahnl — APPROVED** [review]
